src/components/reportPage/ImageResult.jsx

```diff
@@ -16,14 +16,19 @@ const ImageResult = ({ imageFile, imageResult, colourScheme }) => {
 
   const [originalImageDataUrl, setOriginalImageDataUrl] = useState(null);
   const [resultImageDataUrl, setResultImageDataUrl] = useState(null);
+  const [imageName, setImageName] = useState(imageFile.name ?? imageResult.name);
```

Comment:
Not necessary to use useState here, since the file name doesn't change. So this can just be `const imageName = imageFile.name ?? imageResult.name;`